multi-cluster-spring-boot/README.md

```diff
@@ -0,0 +1,174 @@
+# A Sample OpenShift Pipeline for a Spring Boot Application
+
+This example demonstrates how to implement a full end-to-end Jenkins Pipeline for a Java application in OpenShift Container Platform. This sample demonstrates the following capabilities:
+
+* Deploying an integrated Jenkins server inside of OpenShift
+* Running both custom and oob Jenkins slaves as pods in OpenShift
+* "One Click" instantiation of a Jenkins Pipeline using OpenShift's Jenkins Pipeline Strategy feature
+* Promotion of an application's container image within an OpenShift Cluster (using `oc tag`)
+* Promotion of an application's container image to a separate OpenShift Cluster (using `skopeo`)
+* Automated rollout using the [openshift-appler](https://github.com/redhat-cop/casl-ansible/tree/master/roles/openshift-applier) Ansible role.
+
+## Prerequisites
+
+In order to run this pipeline, you will need:
+
+* Two (2) OpenShift clusters, version 3.5 or greater
+  * In this document, we will refer to the first cluster as *_Dev_* and the second as *_Prod_*.
+* Ansible installed on your machine
+
+## Automated Quickstart
+
+This quickstart can be deployed quickly using Ansible. Here are the steps.
+
+1. Clone [this repo](https://github.com/redhat-cop/container-pipelines.git) and the [casl-ansible](https://github.com/redhat-cop/casl-ansible.git) repo.
```

Comment:
To align with the examples in [containers-quickstarts](https://github.com/redhat-cop/containers-quickstarts), let's throw a `requirements.yml` that will bring in any required dependencies